pandas/guide/source/pandas_setup.rst

```diff
@@ -104,7 +104,7 @@ To fetch the latest updates from the pandas repository, follow the steps in
 
 * Create a conda environment: ::
 
-    conda env create -n pandas_dev -f <path-to-pandas-dir>/ci/environment-dev.yaml
+    conda env create -n pandas_dev -f <path-to-pandas-dir>/environment.yaml
```

Comment:
I think that was merged yesterday, you're correct. :)

I think your new url is not correct, as the extension changed from `.yaml` to `.yml`. But with the new change, as the file is the conda default, you can simply install with:

```suggestion
    conda env create
```

Can you give it a try, and update the PR with this if this is correct.

But this has two implications.

First, the name of the environment in the file is `pandas-dev` instead of `pandas_dev`. So, the `source activate pandas_dev` should be updated to use a hyphen.

Then, it is not required anymore to install the optional dependencies (that file does not exist anymore, as the optional dependencies are now in `environment.yml`.

Do you mind making those changes? And thanks for noticing and contributing the fix.